### PR TITLE
object manipulation 2 finally done

### PR DESCRIPTION
**Pull request by Han-Zhang-fuze**
Screenshot 2022-05-23 234002

### CODE REVIEW

File: object-manipulation-2/src/invert.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/* exported invert */
+/*
+convert the object to array using Object.entries method
+swap the each array index
+convert the resulting array to object using Object.fromEntries method
+*/
+function invert(source) {
+  var newArray = Object.entries(source);
+  for (var i = 0; i < newArray.length; i++) {
```
Comment (codingcodymiller) [inline object-manipulation-2/src/invert.js:9]: You can simplify this heavily by just using a `for...in` loop

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: object-manipulation-2/src/invert.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+// function invert(source) {
+//   var newArray = Object.entries(source);
+//   for (var i = 0; i < newArray.length; i++) {
+//     var temp = newArray[i][0];
+//     newArray[i][0] = newArray[i][1];
+//     newArray[i][1] = temp;
+//   }
+//   var newObject = Object.fromEntries(newArray);
+//   return newObject;
+// }
```
Comment (codingcodymiller) [inline object-manipulation-2/src/invert.js:16]: Please remove commented code before submitting work for review.

File: object-manipulation-2/src/invert.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+function invert(source) {
+  var newObject = {};
+  // for (var keys in source) {
+  //   keys = source[keys];
+  //   newObject[keys] = keys;
+  // }
```
Comment (codingcodymiller) [inline object-manipulation-2/src/invert.js:22]: Please remove commented code before submitting work for review.

File: object-manipulation-2/src/omit.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+//         found = true;
+//       }
+//     }
+//     if (!found) {
+//       newObject[key] = source[key];
+//     }
+//     found = false;
+//   }
+//   return newObject;
+// }
```
Comment (codingcodymiller) [inline object-manipulation-2/src/omit.js:38]: Please remove commented code before submitting work for review.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: object-manipulation-2/src/invert.js (hunk 2 above)
Comment (Han-Zhang-fuze) [reply object-manipulation-2/src/invert.js:16]: This code is an alternative solution for the problem, may I keep it for future reference?

File: object-manipulation-2/src/omit.js (hunk 4 above)
Comment (Han-Zhang-fuze) [reply object-manipulation-2/src/omit.js:38]: This code is also an alternative solution for the problem, may I keep it for future reference?

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**